tests/chainerx_tests/unit_tests/routines_tests/test_creation.py

```diff
@@ -1082,6 +1082,10 @@ def test_frombuffer_with_device(device):
 @pytest.mark.parametrize('device', ['native:0', 'cuda:0'])
 @chainerx.testing.parametrize_dtype_specifier('dtype_spec')
 def test_fromfile(xp, count, sep, dtype_spec, device):
+    # Skip if bool_ dtype and text mode
+    if numpy.dtype(dtype_spec) == numpy.bool_ and sep == 'a':
+        return []
```

Comment:
Instead of returning how about calling `pytest.skip`?

Reply:
Maybe it won't work because of the parametrization. In such case is fine to return

Reply:
Okay, I'll take a look.

Reply:
I've fixed it to use imperative `pytest.skip`!

Reply:
Thanks! :)